ui/locales/pl.json

```
@@ -31,17 +31,17 @@
   "PLAY": "Odtwórz",
   "PLAYBACK_RATE": "Szybkość odtwarzania",
   "QUALITY": "Jakość",
-  "RECENTER_VR": "Wycentruj",
-  "REPLAY": "Odtwórz od nowa",
+  "RECENTER_VR": "Wyśrodkuj",
+  "REPLAY": "Odtwórz ponownie",
   "RESOLUTION": "Rozdzielczość",
   "REWIND": "Przewiń do tyłu",
   "SEEK": "Szukaj",
   "SKIP_AD": "Pomiń reklamę",
   "SKIP_TO_LIVE": "Przejdź do transmisji na żywo",
   "STATISTICS": "Statystyki",
-  "SUBTITLE_FORCED": "Wymuszone",
-  "SURROUND": "Przestrzenny",
-  "TOGGLE_STEREOSCOPIC": "Przełącz widok stereoskopowy",
+  "SUBTITLE_FORCED": "Wymuszona",
```

I think it will be better to stick with "Wymuszone" form, as it's plural (the same as Polish word for subtitles, "napisy").
```suggestion
  "SUBTITLE_FORCED": "Wymuszone",
```

---

Reverted.